src/DotLiquid/Context.cs

```diff
@@ -91,6 +93,39 @@ public Context
             SquashInstanceAssignsWithEnvironments();
         }
 
+        /// <summary>
+        /// Creates a new rendering context
+        /// </summary>
+        /// <param name="environments"></param>
+        /// <param name="outerScope"></param>
+        /// <param name="registers"></param>
+        /// <param name="errorsOutputMode"></param>

```

Comment:
Add the XML comment for the new parameter and take the opportunity to fill in the missing comments content for the existing parameters.

Reply:
The XML param declaration has been added but the comment is still empty. Each comment should be something like:

<param name="errorsOutputMode">Errors output mode</param>

(Feel free to expand the descriptions based on your knowledge of what the parameter does).